snippets/partition_options.mdx

```diff
@@ -0,0 +1,25 @@
+- `extract_table_structure`: A boolean that, when `True`, enables DocParse to extract tables and their structural content partitioner using a purpose built table extraction model. If set to `False`, tables are still identified but not analyzed for their structure; as a result, table cells and their bounding boxes are not included in the response. Default is `False`.
+- `table_extraction_options`: A map with string keys specifying options for table extraction, which currently only supports the boolean `include_additional_text`, which will add in text from OCR boxes.  When include_additional_text is set to true and table extraction is enabled, DocParse will attempt to enhance the table structure by merging in tokens from text extraction. This can be useful for working with tables that have missing or misaligned text. `include_additional_text` is `False` by default. The default table_extraction_options is `{}`.
+- `extract_images`: A boolean that determines whether to extract images from the document. Default: `False`.
+- `ocr_images`: A boolean that, when `True`, causes DocParse to use OCR to attempt to generate a text representation of detected images. When `False`, images do not contain a text_representation. Default is `False`.
+- `selected_pages`: A list specifying individual pages (1-indexed) and page ranges from the document to partition. Single pages are specified as integers and ranges are specified as lists with two integer entries in ascending order. A valid example value for selected_pages is `[1, 10, [15, 20]]` which would include pages 1, 10, 15, 16, 17 ..., 20.  `selected_pages` is `None` by default, which results in all pages of the document being parsed.
+- `chunking_options`: A dictionary of options for specifying chunking behavior. Chunking is only performed when this option is present, and default options are chosen when `chunking_options` is specified as `{}`.
+    - `strategy`: A string specifying the strategy to use to combine and split chunks. Valid values are `context_rich`, `mixed_multi_column`, and `maximize_within_limit`. The default and recommended chunker is `context_rich` as `{'strategy': 'context_rich'}`.
+        - Behavior of `context_rich` chunker: The goal of this strategy is to add context to evenly-sized chunks. This is most useful for retrieval based GenAI applications. context_rich chunking combines adjacent `Section-header` and `Title` elements into a new `Section-header` element. Merges elements into a chunk with its most recent `Section-header`. If the chunk would contain too many tokens, then it starts a new chunk copying the Section-header to the start of this new chunk and continues. Merges elements on different pages, unless `merge_across_pages` is set to `False`.
```

Comment:
This is a bit unclear to me. Does this just merge section-header and title or does it merge other elements into section-header as well? I'm not quite sure what's going on with title here. 

"If the chunk would contain too many tokens" is a bit vague, do you mean to say "The strategy will create a chunk that merges elements until the chunk's size  reaches the token limit. A new chunk will be created with the most recently seen section-header copied at the start of the new chunk."

Also can we be clear that the once a new section-header is encountered that new header will be used? 

I feel like a better way of describing this strategy might be: "creates chunks by combining adjacent elements until the chunk reaches the length of the max-token limit specified. Each chunk will contain the copy of the most recently seen section-header" I think this is a bit more concise and captures the sentiment but could still be worded better. We can  discuss in our call.

Reply:
"Consecutive Section-headers and Titles will be treated as a single larger Section-header"

Reply:
- Behavior of `context_rich` chunker: The goal of this strategy is to add context to evenly-sized chunks. Creates chunks by combining adjacent elements until the chunk reaches the length of the max-token limit specified. Each chunk will contain the copy of the most recently seen section-header. Consecutive Section-headers and Titles will be treated as a single larger Section-header.